tests/unit/test_prerequisite.py

```diff
@@ -138,14 +143,79 @@ def test_get_target_points(prereq):
     }
 
 
-def test_get_resolved_dependencies():
+@pytest.fixture
+def prereq2():
     prereq = Prerequisite(IntegerPoint('2'))
     prereq[('1', 'a', 'x')] = True
     prereq[('1', 'b', 'x')] = False
     prereq[('1', 'c', 'x')] = 'satisfied from database'
     prereq[('1', 'd', 'x')] = 'force satisfied'
-    assert prereq.get_resolved_dependencies() == [
+    return prereq
```

Comment:
@wxtim - you'll want to add the strings for skip satisfied dependencies to this list.